### ISSUE TITLE
Npc mass spawn command is broken

### ISSUE THREAD

**Issue by WizardJesse1** [apollo-rsps/apollo#143]
When you try to do `::mass 1 3` it says

```jruby
org.jruby.exceptions.RaiseException:

(ArgumentError) wrong number of arguments (2 for 4) at RUBY.block in (root)(C:\Users\Jess3\Downloads\apollo-master\apollo-master\.\data\plugins\cmd\npc\spawn.rb:57)
at org.jruby.RubyRange.each(org/jruby/RubyRange.java:452) at RUBY.block in (root)(C:\Users\Jess3\Downloads\apollo-master\apollo-master\.\data\plugins\cmd\npc\spawn.rb:56)
at org.jruby.RubyRange.each(org/jruby/RubyRange.java:452) at RUBY.block in (root)(C:\Users\Jess3\Downloads\apollo-master\apollo-master\.\data\plugins\cmd\npc\spawn.rb:55)
at org.jruby.RubyProc.call(org/jruby/RubyProc.java:308) at RUBY.execute(C:\Users\Jess3\Downloads\apollo-master\apollo-master\.\data\plugins\bootstrap.rb:52)
```

**Comment by Major-** [comment]
I've split the two issues that were included into #142 and #143.

**Comment by WizardJesse1** [comment]
alright i did not know if you wanted it broken up or not

**Comment by Major-** [comment]
Fixed in f8db197.
